src/platform/silabs/wifi/wfx_msgs.h

```diff
@@ -26,6 +26,15 @@
 #include "sl_wfx_api.h"
 #include "sl_wfx_constants.h"
 #else
+
+// These names exists in the Si SDK as typedef enum. If they are present in the WF200 builds, we end up with conflicting
+// definitions but no erros because one is a define the other is a typedef enum. This causes different files to use different
+// values.
+#define SL_WFX_STARTUP_IND_ID (1)
```

Comment:
we should rename them at some point

Reply:
Agreed; we need to have a matter abstraction that doesn't re-use all the wf200 structures.